setup.py

```diff
@@ -43,7 +44,6 @@
         'pyyaml',
         'requests',
         'scitools-iris>=3.0.1',
-        'requests',
```

Comment:
Why are you removing this dependency? It is used in `esmvalcore/_citation.py`.

Reply:
I noticed it's in the list twice (line 44) while I was adding the ninja2 dependency 😅